less/navigation.less

```diff
@@ -1 +1,59 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.grey-banner{
+    width: 100%;
+    background: #f4f4f4;
+    border-bottom: 2px solid rgb(235, 233, 233);
+   
+}
+
+@media @mobile{
+    .grey-banner{
+    background: white;
+    border: none;
+    width: 100%;
+}}
+
+.navbar{
+    // border: 1px solid blue;
+    width: 800px;
+    height: 10rem;
+    margin: 0 auto;
+    @media @mobile{
+        width:100%
+    }
+
+    h1{
+        // border: 1px solid yellow;
+        .center();
+        width: 100%;
+        height: 6rem;
+        font-size: 4.2rem;
+        @media @mobile{
+            font-size: 3rem;
+        }
+    }
+
+    nav {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        margin: 1% 24% 0;
+        @media @mobile{
+            margin: 1% 7% 0%;
+            font-size: 70%;
+        }
+    
+        // border: 1px solid red;
+
+        a {
+            // border: 1px solid green;
+            .text-blank();
+            font-size: 18px;
+            @media @mobile{
+                font-size: 1rem;
+            }
```

Comment:
Setting pixels in desktop causes your links to be non responsive, wrap and collide with other links in the nav bar. Try using rems instead.

For the mobile media query, using rems is great but it seems like 1rem might be too small.